javascript-timers/timeout.js

```diff
@@ -0,0 +1,2 @@
+var messageElement = document.querySelector('.message');
+messageElement.textContent = 'Hello There';
```

Comment:
This isn't making use of `setTimeout` at all

Reply:
![Kapture 2022-04-14 at 14 40 42](https://user-images.githubusercontent.com/99709530/163481404-c5cc03ee-a3e6-4d56-8623-76027c52fcde.gif)